doc/api.apib

```diff
@@ -174,11 +183,342 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 + Request (application/json)
 
         {
-            "docker_image": "busybox:latest"
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+        }
```

Comment:
Unsure if the component level actions were ever required by anyone? I think if usages are extremely rare then I'd prefer to not maintain codebase for setting state at component level at all.